test/performance/README.md

```diff
@@ -1,10 +1,81 @@
 # Performance tests
 
-Knative performance tests are test geared towards producing useful performance
+Knative performance tests are tests geared towards producing useful performance
 metrics of the knative system. As such they can choose to take a blackbox
 point-of-view of the system and use it just like an end-user might see it. They
 can also go more whiteboxy to narrow down the components under test.
 
-Existing scenarios:
+## Adding performance tests
 
-- [Observed concurrency](./observed-concurrency)
+Performance tests are simple go tests, that use [Fortio](#load-generator) for generating 
+load and [Tesgrid](#displaying-metrics) to see the metrics on a continous basis. For whitebox tests,
+the performance tests also bring up the prometheus service alongwith the knative-serving. So, 
+you can [query prometheus](#prometheus-metrics) to get server side metrics with the load 
+generator metrics.
+
+## Load Generator
+
+Knative has a [load generator library](https://github.com/knative/test-infra/tree/master/shared/loadgenerator)
+that can be used to generate load. The load generator uses [Fortio](https://fortio.org/) for generating load based
+on the [generator options](https://github.com/knative/test-infra/blob/master/shared/loadgenerator/loadgenerator.go#L36)
```

Comment:
I suggest not linking to specific lines, as they will quickly point to the wrong place once you update the code.